code/modules/projectiles/ammo_datums.dm

```diff
@@ -47,6 +47,12 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	var/hitscan_effect_icon = "beam"
 	///A multiplier applied to piercing projectile, that reduces its damage/penetration/sundering on hit
 	var/on_pierce_multiplier = 1
+	///greyscale settings for the bullet items associated with the ammo
+	var/handful_greyscale_config = null
+	var/handful_greyscale_colors = null
+	///greyscale settings for the projectile associated with the ammo
+	var/projectile_greyscale_config = null
+	var/projectile_greyscale_colors = null
```

Comment:
comment all vars pls (copy pasting the comment is fine)

Reply:
done.